analyses/interaction-plots/scripts/cooccur_functions.R

```diff
@@ -131,14 +151,22 @@ coocurrence <- function(gene_sample_df,
       mut00 = sum(muts1 == 0 & muts2 == 0),
       odds_ratio = (mut11 * mut00) / (mut10 * mut01),
       cooccur_sign = ifelse(odds_ratio > 1, 1, -1)
-    ) %>%
+      ) %>%
     dplyr::rowwise() %>%
     dplyr::mutate(p = row_fisher(mut11, mut10, mut01, mut00)) %>%
     dplyr::ungroup() %>%
     dplyr::mutate(
       q = p.adjust(p, method = "BH"),
-      cooccur_score = cooccur_sign * -log10(p)
-    )
-
+      cooccur_score = cooccur_sign * -log10(p)) %>%
+    dplyr::rowwise() %>%
+    dplyr::mutate(    
+      n_mutated_gene1 = sum(mut11, mut10),
+      n_mutated_gene2 = sum(mut11, mut01),
+      perc_mutated_gene1 = sum(mut11, mut10)*100/sum(mut11, mut10, mut01, mut00),
+      perc_mutated_gene2 = sum(mut11, mut01)*100/sum(mut11, mut10, mut01, mut00),
+      perc_cooccur = mut11*100/sum(mut11 + mut10 + mut01),
```

Comment:
Just making this parallel with the other calculations here.
```suggestion
      perc_cooccur = mut11*100/sum(mut11, mut10, mut01),
```